static/js/tabs.js

```diff
@@ -0,0 +1,44 @@
+function initTabs() {
+  var tabLinks = document.querySelectorAll("a[href^='#']");
+
+  // prevent the page jumping around when switching tabs,
+  // whilst still using :target
+  // https://gist.github.com/pimterry/260841c2104f27cadc954a29b9873b96#file-disable-link-jump-with-workaround-js
+  [].forEach.call(tabLinks, function(link) {
+    link.addEventListener("click", function(event) {
+      event.preventDefault();
+      history.pushState({}, "", link.href);
+
+      // Update the URL again with the same hash, then go back
+      history.pushState({}, "", link.href);
+      history.back();
+
+      setActiveTab();
+    });
+  });
+
+  document.addEventListener("DOMContentLoaded", setActiveTab());
+
+  window.addEventListener(
+    "hashchange",
+    function() {
+      setActiveTab();
+    },
+    false
+  );
+
+  function setActiveTab() {
```

Comment:
as a personal preference I like to pre-hoist the functions so you don't have to scroll through the file looking for content that's not in a function.